src/main/java/edu/cornell/kfs/concur/businessobjects/ValidationResult.java

```diff
@@ -0,0 +1,52 @@
+package edu.cornell.kfs.concur.businessobjects;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ValidationResult {
+
+    protected boolean valid;
+    protected List<String> messages;
+
+    public ValidationResult(boolean valid, List<String> messages) {
+        this.valid = valid;
+        this.messages = messages;
+    }
+
+    public boolean isValid() {
+        return valid;
+    }
+
+    public void setValid(boolean valid) {
+        this.valid = valid;
+    }
+
+    public List<String> getMessages() {
+        return messages;
+    }
+
+    public void setMessages(List<String> messages) {
+        this.messages = messages;
+    }
+
+    public void addMessage(String message) {
+        if (messages == null) {
+            messages = new ArrayList<String>();
+        }
+        messages.add(message);
+    }
+
+    @Override
+    public String toString() {
+        StringBuffer result = new StringBuffer();
```

Comment:
I have checked the specs and there is no message specified for the case when the account info is valid. When account info is valid the next step is to approve the Expense Report. I have sent the question to Sherry to see if a message would be needed. I can update this when the account validation with error messages is implemented in Sprint 16 if there is a need for it.